src/rgw/rgw_acl_s3.cc

```diff
@@ -489,6 +490,7 @@ int RGWAccessControlPolicy_S3::rebuild(RGWUserCtl *user_ctl, ACLOwner *owner, RG
   RGWUserInfo owner_info;
   if (user_ctl->get_info_by_uid(owner->get_id(), &owner_info, null_yield) < 0) {
     ldout(cct, 10) << "owner info does not exist" << dendl;
```

Comment:
probably better to unify the log message with the error message sent in the reply. put the string into the error message and then pass it also to ldout

Reply:
I think the log message is used to help developers to find the issues and bugs in code, but the error message returned by http server is used to tell user what happens and why returns an error response.

So, I think it is not necessary to unify the log message with the error message, the log message should contain more detailed information to help developers to solve issues.

Reply:
> I think the log message is used to help developers to find the issues and bugs in code, but the error message returned by http server is used to tell user what happens and why returns an error response.
> 
> So, I think it is not necessary to unify the log message with the error message, the log message should contain more detailed information to help developers to solve issues.

this is true for the log message in line 536 (you don't want to put function names in the HTTP reply...) but in the other cases, the log and error message are pretty much the same, only differently phrased.

Reply:
> > I think the log message is used to help developers to find the issues and bugs in code, but the error message returned by http server is used to tell user what happens and why returns an error response.
> > So, I think it is not necessary to unify the log message with the error message, the log message should contain more detailed information to help developers to solve issues.
> 
> this is true for the log message in line 536 (you don't want to put function names in the HTTP reply...) but in the other cases, the log and error message are pretty much the same, only differently phrased.

I think we should keep the response error message same with AWS S3 for the compatibility. What do you think about it? @cbodley

Reply:
@ZVampirEM77 if this is a matter of compatibility then I agree. More important than internal consistency